unit-tests/py/rspy/libci.py

```diff
@@ -130,13 +134,13 @@ class TestConfigFromText( TestConfig ):
         //#test:...
     """
 
-    def __init__( self, source, line_prefix ):
+    def __init__( self, source, line_prefix, context = None ):
```

Comment:
I don't think it should have a default -- if you're loading text, you need to consciously decide on a context

Reply:
(It can have a default in Test, but not in TestConfigFromText)